src/main/java/com/spartronics4915/frc2020/subsystems/LED.java

```diff
@@ -22,7 +22,7 @@ public static LED getInstance()
         if (sInstance == null)
         {
             sInstance = new LED();
-            mBlingState = BlingState.OFF;
+            mBlingState = BlingState.BLING_COMMAND_STARTUP;
```

Comment:
@MayaShaw @randomgrace Pls make sure the initial bling state is the same on both parties. Arduino code currently is set to `currentCommand = OFF;` ([see](https://github.com/MayaShaw/2020-InfiniteRecharge/blob/dc594860d48a9109825dcc40a50403a3355f7193/Arduino-Bling/RobotBling/RobotBling.ino#L113)). Otherwise, one will just stump over the other, and could cause confusion on what was expected at initialization of both systems.

Reply:
Actually, this is a mute point and doesn’t matter. Pls ignore — sorry for the nosie!